daemon/entrypoint.sh

```diff
@@ -276,7 +276,12 @@ function osd_directory {
   fi
 
   for OSD_ID in $(ls /var/lib/ceph/osd |  awk 'BEGIN { FS = "-" } ; { print $2 }'); do
-    if [ -n "${JOURNAL_DIR}" ]; then
+    OSD_JOURNAL_TMP="OSD_JOURNAL$OSD_ID"
+    OSD_JOURNAL=${!OSD_JOURNAL_TMP}
+    if [ -n "${OSD_JOURNAL}" ]; then
+      ceph-disk -v prepare ${OSD_JOURNAL}
```

Comment:
I would greatly prefer to not corrupt `osd_directory` mode with that `ceph-disk` script.  You can pass a device to the `ceph-osd ... --mkjournal --osd-journal ...` command, so there should be no need for `ceph-disk`.